lib/ChainwebDb/Types/Transaction.hs

```diff
@@ -1,44 +1,41 @@
-{-# OPTIONS_GHC -fno-warn-missing-signatures #-}
-{-# LANGUAGE DeriveAnyClass #-}
-{-# LANGUAGE DeriveGeneric #-}
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE FlexibleInstances #-}
-{-# LANGUAGE GADTs #-}
-{-# LANGUAGE ImpredicativeTypes #-}
-{-# LANGUAGE MultiParamTypeClasses #-}
-{-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE StandaloneDeriving #-}
-{-# LANGUAGE TemplateHaskell #-}
-{-# LANGUAGE TypeApplications #-}
-{-# LANGUAGE TypeFamilies #-}
-{-# LANGUAGE TypeSynonymInstances #-}
+{-# LANGUAGE DeriveAnyClass       #-}
+{-# LANGUAGE DeriveGeneric        #-}
+{-# LANGUAGE DerivingStrategies   #-}
+{-# LANGUAGE FlexibleContexts     #-}
+{-# LANGUAGE FlexibleInstances    #-}
+{-# LANGUAGE ImpredicativeTypes   #-}
+{-# LANGUAGE StandaloneDeriving   #-}
+{-# LANGUAGE TypeFamilies         #-}
 {-# LANGUAGE UndecidableInstances #-}
 
+{-# OPTIONS_GHC -fno-warn-missing-signatures #-}
+
 module ChainwebDb.Types.Transaction where
 
 ------------------------------------------------------------------------------
-import           Data.Aeson
-import           Data.Text (Text)
-import           Data.Time
-import           Database.Beam
+import Data.Aeson
+import Data.Text (Text)
+import Data.Time
+import Database.Beam
 ------------------------------------------------------------------------------
-import           ChainwebDb.Types.Block
+import ChainwebDb.Types.Block
 ------------------------------------------------------------------------------
 
 
 ------------------------------------------------------------------------------
 data TransactionT f = Transaction
-  { _transaction_id :: C f Int
-  , _transaction_chainId :: C f Int
-  , _transaction_block :: PrimaryKey BlockT f
+  { _transaction_id           :: C f Int
```

Comment:
Let's avoid this vertical alignment for fields in a data type.  It makes diffs larger and increases the number of merge conflicts.